CHANGELOG.md

```diff
@@ -17,6 +17,13 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
   [hovox](https://github.com/hovox)
   [#8797](https://github.com/CocoaPods/CocoaPods/issues/8797)
 
+##### Enhancements
```

Comment:
I think this is the wrong entry.

Reply:
I will fix.

Reply:
dang, got messed up during a rebase. my bad!